### PR TITLE
Update spikeglx and neuralynx

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@luiztauffer Updating the conversion process for this project similar to your previous suggestions of having a different converter per experiment type.

Each experiment has "raw" (ElectricalSeries - in this PR, Neuralynx and SpikeGLX) and "processed" (trial and events data, some previous spike sorting occasionally).

Trial times are set as `np.nan` until the lab gets back about a good way to actually calculate/extract those.

### CODE REVIEW

File: brody_lab_to_nwb/convert_brody/brodynwbconverter.py [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-class BrodysNWBConverter(NWBConverter):
-    """Primary conversion class for the Brody lab processing pipeline."""
+class PoissonClicksNWBConverter(NWBConverter):
+    """Primary conversion class for the SpikeGLX formatted Brody lab data."""
 
     data_interface_classes = dict(
-        SpikeGadgetsRecording=SpikeGadgetsRecordingInterface,
-        NeuralynxRecording=NeuralynxRecordingInterface,
         SpikeGLXRecording=SpikeGLXRecordingInterface,
         SpikeGLXLFP=SpikeGLXLFPInterface,
```
Comment (luiztauffer) [inline brody_lab_to_nwb/convert_brody/brodynwbconverter.py:14]: @CodyCBakerPhD are there separate files for raw and lfp in this case?

Comment (CodyCBakerPhD) [reply brody_lab_to_nwb/convert_brody/brodynwbconverter.py:14]: Yep. Standard SpikeGLX output and actively choosing to write both here.

Comment (luiztauffer) [reply brody_lab_to_nwb/convert_brody/brodynwbconverter.py:14]: I think [these changes](https://github.com/catalystneuro/nwb-conversion-tools/pull/189) will be required for this to work

Comment (luiztauffer) [reply brody_lab_to_nwb/convert_brody/brodynwbconverter.py:14]: see also https://github.com/catalystneuro/movshon-lab-to-nwb/blob/c1cac6426a6fc6c31b7eee7d8a9bb23281bc8331/movshon_lab_to_nwb/movshonnwbconverter.py#L43-L48

**Pull request merged**